### PR TITLE
core: truncate measure timings to hundredths

### PR DESCRIPTION
**Pull request by Hoten**
Reduces LHR size by ~4KB. That was ~1% of the LHR I tested it with.

**Related Issues/PRs**

#7160

### CODE REVIEW

**Comment by Hoten** [comment]
Wait, wrong timings :)

File: lighthouse-core/runner.js [hunk 1]
```diff
@@ -180,6 +180,12 @@ class Runner {
       // As entries can share a name, dedupe based on the startTime timestamp
     ].map(entry => /** @type {[number, PerformanceEntry]} */ ([entry.startTime, entry]));
     const timingEntries = Array.from(new Map(timingEntriesKeyValues).values());
+    for (const timing of timingEntries) {
```
Comment (paulirish) [inline lighthouse-core/runner.js]: add a comment above this loop to explain why this is being done.

File: lighthouse-core/runner.js [hunk 2]
```diff
@@ -180,6 +180,12 @@ class Runner {
...
+      // @ts-ignore - ignore readonly
+      timing.startTime = parseFloat(timing.startTime.toFixed(2));
```
Comment (brendankenny) [inline lighthouse-core/runner.js]: It looks like you may have to make a copy of these in order to do this. Real `PerformanceEntry`'s properties really do appear to be read only

Comment (paulirish) [reply lighthouse-core/runner.js]: yeaup. 
image

Comment (Hoten) [reply lighthouse-core/runner.js]: done

**Comment by paulirish** [comment]
shout out to @wardpeet 	who made extension-test a thing!  that failure was so rad. :D

**Comment by patrickhulce** [comment]
hip hip hurray!! @wardpeet is like, *the* best example of open source awesomeness around 😃

**Pull request merged**